DFE-Digital/early-careers-framework
# [CPDLP-1175] Refactor ECF statements + new designs

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-1175

- Tweak ECF statement to conform-ish to new designs
- Tweaks to NPQ statement to handle mobile views better
- Removed many view components as there seemed to be many for what was needed
- Introduced `StatementCalculator` as to hang off calculations for a statement
- Concept of `EligibleAggregator` still remains until auto declaration assignment is merged
- ECF statements should now be able to handle all available milestone declarations
- Contract information now shown inline inside an expandy box thing as opposed to a different page

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- Login as a finance user
- View an ECF statement
- Should see new designs

## External API changes

- None
